docs/src/main/tut/docs/core/modules/README.md

```diff
@@ -40,8 +45,8 @@ import algebras._
 
 object modules {
     @module trait Persistence {
-      val database: Database
-      val cache: Cache
+      val database: Database.StackSafe
```

Comment:
There is modules for tagless and therefore this isn't needed anymore to use the StackSafe representation to embed them into modules